doc/introduction/pulse.rst

```diff
@@ -0,0 +1,289 @@
+    H1 =  2 * XX + f1 * YY + f2 * ZZ
+
+    # Option 2
+    coeffs = [2, f1, f2]
+    ops = [XX, YY, ZZ]
+    H2 =  qml.ops.dot(coeffs, ops)
+
+
+.. warning::
+    When initializing a :class:`~.ParametrizedHamiltonian` via a list of parametrized coefficients, it
+    is possible to create a list of multiple coefficients of the same form iteratively using lambda
+    functions, i.e.
+
+    ``coeffs = [lambda p, t: p for _ in range(3)]``.
+
+    Do **not**, however, define the function as dependent on the value that is iterated over. That is, it is not
+    possible to define ``coeffs = [lambda p, t: p * t**i for i in range(3)]`` to create a list
+    ``coeffs = [(lambda p, t: p), (lambda p, t: p * t), (lambda p, t: p * t**2)]``. The value of ``i`` when
+    creating the lambda functions is set to be the final value in the iteration, such that this will
+    produce ``coeffs = [(lambda p, t: p * t**2)] * 3``.
+
+
+The :class:`~.ParametrizedHamiltonian` is a callable, and can return an :class:`~.Operator` if passed a set of
+parameters and a time at which to evaluate the coefficients :math:`f_j`.
+
+.. code-block:: python
+
+    >>> H1
+    ParametrizedHamiltonian: terms=3
+    >>> params = [1.2, [2.3, 3.4]]  # f1 takes a single parameter, f2 takes 2
+    >>> H1(params, t=0.5)
+    (2*(PauliX(wires=[0]) @ PauliX(wires=[1]))) + ((-0.2876553535461426*(PauliY(wires=[0]) @ PauliY(wires=[1]))) + (1.5179612636566162*(PauliZ(wires=[0]) @ PauliZ(wires=[1]))))
+    >>> qml.equal(H1(params, t=0.5), H2(params, t=0.5))
+    True
+
+.. warning::
+    Order matters here. When initializing the :class:`~.ParametrizedHamiltonian`, terms defined with fixed coefficients
+    should come before parametrized terms to prevent discrepancy in the wire order. When passing parameters, ensure
+    that the order of the coefficient functions and the order of the parameters match.
+
+Convenience functions for building a ParametrizedHamiltonian
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The convenience functions currently available in PennyLane to assist in creating coefficients functions
+for a :class:`~.ParametrizedHamiltonian` are:
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.pulse.constant
+    ~pennylane.pulse.pwc
+    ~pennylane.pulse.pwc_from_function
+    ~pennylane.pulse.rect
+
+:html:`</div>`
+
+Further examples
+^^^^^^^^^^^^^^^^
+
+A few additional examples of defining a :class:`~.ParametrizedHamiltonian` are provided here.
+
+Using ``rect`` to create a parametrized coefficient that has a value of 0 outside the time interval
+:math:`t=(1, 7)`, and within the interval is defined by ``jnp.polyval(p, t)``:
+
+.. code-block:: python
+
+    from pennylane.pulse.convenience_functions import rect
+
+    def f(p, t):
+        return jnp.polyval(p, t)
+    H = qml.pulse.rect(f1, windows=[(1, 7)]) * qml.PauliX(0)
+
+    # inside the window
+    H([3], t=2)
+    >>> 2.7278921604156494*(PauliX(wires=[0]))
```

Comment:
Is this a standard formatting with the `>>>` on the output lines instead of the input lines? I only know it the other way around (like you used further up):
```suggestion
    >>> H([3], t=2)
    2.7278921604156494*(PauliX(wires=[0]))
```

Reply:
I don't think so 😆